queries/buildings.jinja2

```diff
@@ -55,6 +52,7 @@ UNION ALL
 SELECT
     name,
     NULL AS area,
+    'addr' AS kind,
     osm_id AS __id__,
     NULL AS "building:part",
     NULL AS building,
```

Comment:
Both sides of a `UNION` must have the same columns, so you'll need to remove "building", "amenity", "shop" and "tourism" from the address points as well.